src/diffpy/labpdfproc/tests/conftest.py

```diff
@@ -36,6 +36,8 @@ def user_filesystem(tmp_path):
     with open(input_dir / "binary.pkl", "wb") as f:
         f.write(binary_data)
 
+    with open(base_dir / "file_list.txt", "w") as f:
+        f.write("./*.txt")
     with open(input_dir / "file_list.txt", "w") as f:
```

Comment:
This is fine, though we could have tested it with a single file_list.txt file with the right contents.